handler/category-dimension-import.go

```diff
@@ -118,6 +126,8 @@ func (h *CategoryDimensionImport) Handle(ctx context.Context, e *event.CategoryD
 						return err
 					}
 					i-- // retry this iteration with new eTag value, as the instance is still in a valid state
+					SleepRandom(attempt)
+					attempt++
```

Comment:
If for some reason a race condition develops with another instance of this app or there is some other bug / problem ...
There needs to be a check on how large 'attempts' has grown, say '10' for example, and when its greater than '10' the retries need to stop and some other error is logged and an error is returned.

Reply:
yes, good point. I think this scenario is unlikely, but we need to handle it. I will set a maxAttempts, abort and try to set instance to failed state.